package.json

```diff
@@ -18,20 +18,20 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^11.2.4",
-    "@angular/cdk": "^11.2.3",
-    "@angular/common": "^11.2.4",
-    "@angular/compiler": "^11.2.4",
-    "@angular/core": "^11.2.4",
+    "@angular/animations": "^11.2.8",
```

Comment:
any reason for going to 11.2.8 rather than 11.2.11? (https://github.com/angular/angular/blob/master/CHANGELOG.md#11211-2021-04-21)

Reply:
Currently there are some incompatibility problems with storybook